tests/Unit/TracingFactoryTest.php

```diff
@@ -2,12 +2,15 @@
 
 namespace ZipkinBundle\Tests\Unit;
 
-use PHPUnit_Framework_TestCase;
 use Psr\Log\NullLogger;
-use Symfony\Component\DependencyInjection\Container;
-use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
+use PHPUnit_Framework_TestCase;
```

Comment:
Should you use this deprecated class? Or is better to use `PHPUnit\Framework\TestCase` class instead?

Reply:
It would be great if we can move everything to the namespace classes or even better [upgrade phpunit version](https://github.com/jcchavezs/zipkin-instrumentation-symfony/issues/48). Are you up for a PR?